dotnet/src/Experimental/Process.Runtime.Dapr/DaprProcessInfo.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// A serializable representation of a Dapr Process.
+/// </summary>
+[KnownType(typeof(KernelProcessEdge))]
+[KnownType(typeof(KernelProcessState))]
+[KnownType(typeof(KernelProcessStepState))]
+[KnownType(typeof(KernelProcessStepState<>))]
+public sealed record DaprProcessInfo : DaprStepInfo
```

Comment:
This is required to be public because it's used in a Dapr Actor Interface which are required to be public.